src/services/sessionInvite.js

```diff
@@ -25,7 +25,47 @@ const cacheHelper = require('@generics/cacheHelper')
 module.exports = class UserInviteHelper {
 	static async uploadSession(data) {
 		return new Promise(async (resolve, reject) => {
+			const startTime = Date.now()
+			const jobId = `${data.fileDetails.id}_${startTime}`
+
 			try {
+				// Validate defaults early before processing begins
+				let defaults = null
+				try {
+					defaults = await getDefaults()
+				} catch (defaultsError) {
+					// Use environment variable defaults as fallback
+					defaults = {
+						orgCode: process.env.DEFAULT_ORGANISATION_CODE || 'default_code',
+						tenantCode: process.env.DEFAULT_TENANT_CODE || 'default',
+					}
+				}
+
+				if (!defaults || !defaults.orgCode) {
+					// Update file upload status to FAILED before throwing error
+					try {
+						await fileUploadQueries.update({ id: data.fileDetails.id }, data.user.tenant_code, {
+							status: common.STATUS.FAILED,
+							updated_by: data.user.userId,
+						})
+					} catch (dbError) {
+						console.error('Failed to update file upload status:', dbError)
+					}
+					throw new Error('DEFAULT_ORG_CODE_NOT_SET')
+				}
+				if (!defaults.tenantCode) {
```

Comment:
@sumanvpacewisdom we wil have default tenant code and default org code right, then why this check ?